app/javascript/packs/stores/main_store.js

```diff
@@ -1,20 +1,52 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import types from '../mutations/main_mutations_types'
+import vuejsStorage from 'vuejs-storage'
 Vue.use(Vuex)
+
 const state = {
-  default_url: "https://validato-va-34e2.mainnet.rpcpool.com",
-  localhost_url: "https://api.mainnet-beta.solana.com"
+  mainnet_url: "https://validato-va-34e2.mainnet.rpcpool.com",
+  testnet_url: "https://api.testnet.solana.com"
 }
 const getters = {
-  web3_url: function(state) {
-    if(location.hostname === "localhost"){
-      return state.localhost_url
+  web3_url: function(state, getters) {
+    if(location.hostname === "localhost" || getters.network === 'testnet'){
```

Comment:
we often need to test mainnet on localhost as well, so update the ifs to capture all possibilities.

Reply:
Fixed.